### DISCUSSION TITLE
Import Airtable to Postgres issue [May 30]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2274] [Discord Threads]
> 🙋‍♂️ **mahmouds12**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### https://github.com/nocodb/nocodb/issues/2199   
####    
#### Just clarifying, had you clicked "Preview > Viewer" from right top menu ?  
#  
> 💁 **o1lab**  
#### <@847339595314364467>  
#  
> 🙋‍♂️ **mahmouds12**  
#### <@310157751395287040> no  
#  
> 🙋‍♂️ **mahmouds12**  
#### I tried another airtable base, and it didn't have the same error  
#  
> 💁 **o1lab**  
#### ahh, u mean the migration was fine  
#  
> 🙋‍♂️ **mahmouds12**  
#### I mean it didn't work with one airtable base but worked for another one  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/980535707233828895)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
